string related_class           # ex. "person", "chair"
float32 score                  # [0, 1]
geometry_msgs/Pose pose        # center pose in map frame (z ~= 0)
float32 width                  # [m]
float32 height                 # [m]